### PR TITLE
pythonPackages.xml2rfc: fix build

### PR DESCRIPTION
**Pull request by Ma27**
###### Motivation for this change

Fixes the broken `xml2rfc` package by adding several python libraries to the build inputs list. The dependency `google-i18n-address` had to be packaged.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/google-i18n-address/default.nix [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    inherit pname version;
+    sha256 = "0f1j1lp9bmllkzhciw0lxi7ipm8w461n0p97mz9714br0cs9glm1";
+  };
+
+  propagatedBuildInputs = [ requests ];
+
+  checkInputs = [ pytest pytestcov mock ];
+
+  meta = with lib; {
+    description = "Google's i18n address data packaged for Python.";
```
Comment (worldofpeace) [inline pkgs/development/python-modules/google-i18n-address/default.nix]:
```suggestion
    description = "Google's i18n address data packaged for Python";
```

**Comment by worldofpeace** [comment]
Seeing warnings like
```
WARNING: Unable to find a suitible cache directory to write to, trying the following directories:
  /var/cache/xml2rfc
  /homeless-shelter/.cache/xml2rfc
Try giving a specific directory with --cache.
```
when the tests are ran. Not sure if it's important.

Also seeing `six` in `requirements.txt` and it's missing here.

**Comment by Ma27** [comment]
> when the tests are ran. Not sure if it's important.

Could be fixed by setting `HOME` to a temporary directory in the check phase.

> Also seeing six in requirements.txt and it's missing here.

Good catch! Not exactly sure why it built successfully, but I added this for now :)

**Comment by worldofpeace** [comment]
@GrahamcOfBorg build pythonPackages.xml2rfc python3Packages.xml2rfc

**Pull request merged**